spec/jobs/import_url_job_spec.rb

```diff
@@ -13,10 +17,10 @@
   end
 
   let(:operation) { create(:operation) }
-  let(:actor) { instance_double(Hyrax::Actors::FileSetActor, create_content: true) }
+  let(:actor) { instance_double(JobIoWrapper, ingest_file: true) }
```

Comment:
Is `actor` a misleading name for this variable now that it is `JobIoWrapper` and not an "actor"?

Reply:
Yeah, good call.